Makefile

```diff
@@ -0,0 +1,41 @@
+.PHONY: build test clean prepare update docker
+
+GO = CGO_ENABLED=0 GO111MODULE=on go
+
+MICROSERVICES=cmd/device-gpio-go
+
+.PHONY: $(MICROSERVICES)
+
+DOCKERS=docker_device_gpio_go
+.PHONY: $(DOCKERS)
+
+VERSION=$(shell cat ./VERSION 2>/dev/null || echo 1.0.0)
+GIT_SHA=$(shell git rev-parse HEAD)
+GOFLAGS=-ldflags "-X github.com/edgexfoundry/device-gpio-go.Version=$(VERSION)"
+
+build: $(MICROSERVICES)
+	$(GO) build ./...
```

Comment:
This line is not needed. Cause double build. Not in the device-rest makefile and needs to be cleaned up in other device service makefiles.

Reply:
This still needs to be addressed